cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90

```diff
@@ -1211,11 +1216,50 @@ subroutine ModelAdvance(gcomp, rc)
     !--------------------------------
     ! Advance cice and timestep update
     !--------------------------------
-
-    if(profile_memory) call ESMF_VMLogMemInfo("Entering CICE_Run : ")
-    call CICE_Run()
-    if(profile_memory) call ESMF_VMLogMemInfo("Leaving CICE_Run : ")
-
+    !call ESMF_LogWrite('PRE CICE_RUN', ESMF_LOGMSG_INFO) 
```

Comment:
@SmithJos13 Again, why this is removed. It is just profiling memory. We are not trying improve or modify the code. The only think that we are trying is to add new capability with minimal code change. If we have these changes and not to push CICE consortium or UFS, we will diverge and it would be hard for us to sync CICE in the future.